### PR TITLE
11 page

### PR DESCRIPTION
**Pull request by soysreyvy**

### CODE REVIEW

**Review by kimleangkheng — COMMENTED** [review]
All well and good

File: developer/cloud/3.0/index.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+|-------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| GET [https://api.keyman.com/cloud/3.0/languages](https://api.keyman.com/cloud/3.0/languages){target="_blank"}                                   | Returns a CloudServices object with a Languages object, that contains a list of all language and keyboard combinations available on the server. This call will return full details of every language, keyboard and font combination. It is typically around 150KB. |
```
Comment (jahorton) [inline developer/cloud/3.0/index.md:23]: We have a few `{target="_blank"}` entries on links.  It'd be nice if they work, but we'll have to remove them if they show up in plain-text when loaded on the site.

Comment (jahorton) [reply developer/cloud/3.0/index.md:23]: And, unfortunately...

image

Comment (jahorton) [reply developer/cloud/3.0/index.md:23]:
```suggestion
| GET [https://api.keyman.com/cloud/3.0/languages](https://api.keyman.com/cloud/3.0/languages)                                   | Returns a CloudServices object with a Languages object, that contains a list of all language and keyboard combinations available on the server. This call will return full details of every language, keyboard and font combination. It is typically around 150KB. |
```

File: developer/cloud/3.0/index.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+| GET [https://api.keyman.com/cloud/3.0/keyboards/**european**](https://api.keyman.com/cloud/3.0/keyboards/european){target="_blank"}             | Returns a CloudServices object with a Keyboard object for the keyboard identified by id **european**, and a list of all languages associated with that keyboard.                                                                                                   |
+| GET [https://api.keyman.com/cloud/3.0/keyboards/**european**/**fra**](https://api.keyman.com/cloud/3.0/keyboards/european/fra){target="_blank"} | Returns a CloudServices object with a Keyboard object for the keyboard identified by id **european**, and the language associated with **fra**.                                                                                                                    |
```
Comment (jahorton) [inline developer/cloud/3.0/index.md:28]:
```suggestion
| GET [https://api.keyman.com/cloud/3.0/languages/**fra**](https://api.keyman.com/cloud/3.0/languages/fra)                       | Returns a CloudServices object with a Language object for a language identified by the language code **fra**.                                                                                                                                                      |
| GET [https://api.keyman.com/cloud/3.0/languages/**fra/european**](https://api.keyman.com/cloud/3.0/languages/fra/european)     | Returns a CloudServices object with a Language object for a language identified by the language code **fra** and its associated keyboard identified by id **european**.                                                                                            |
| GET [https://api.keyman.com/cloud/3.0/keyboards](https://api.keyman.com/cloud/3.0/keyboards)                                   | Returns a CloudServices object with a Keyboards object that contains a list of all keyboards on the server. The response is typically around 75KB.                                                                                                                 |
| GET [https://api.keyman.com/cloud/3.0/keyboards/**european**](https://api.keyman.com/cloud/3.0/keyboards/european)             | Returns a CloudServices object with a Keyboard object for the keyboard identified by id **european**, and a list of all languages associated with that keyboard.                                                                                                   |
| GET [https://api.keyman.com/cloud/3.0/keyboards/**european**/**fra**](https://api.keyman.com/cloud/3.0/keyboards/european/fra) | Returns a CloudServices object with a Keyboard object for the keyboard identified by id **european**, and the language associated with **fra**.                                                                                                                    |
```

**Review by jahorton — APPROVED** [review]

**Pull request merged**